src/lib/components/FormFeedback.js

```diff
@@ -57,12 +57,13 @@ class DisconnectedFormFeedback extends Component {
    * @returns String or React node
    */
   renderErrorMessages(messages) {
-    if (messages.length === 1) {
+    let uniqueMessages = messages.filter((v, i, a) => a.indexOf(v) === i);
```

Comment:
Cool, I was thinking [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set) but that works :+1: .

Reply:
@jrcastro2 I would go for either Set as @fenekku suggested or at least https://lodash.com/docs/4.17.15#uniqBy . It's much clearer I think the intention of what we try to achieve here.....

Reply:
Yes, changed to Set, thanks!

P.S: `Additional Descriptions` added.